WebUI/Data/Models/Review.cs

```diff
@@ -24,7 +24,7 @@ public partial class Review
         [StringLength(512)]
         public string Description { get; set; }
         [Column("approved")]
-        public bool Approved { get; set; }
+        public bool? Approved { get; set; }
```

Comment:
This change makes sense, but you need to support it by creating a migration as it affects the DB.

Reply:
I've added the migration I was having issues with the command line but I was able to fix it